compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -698,14 +740,29 @@ def IREELinalgExt_OnlineAttentionOp : IREELinalgExt_PureOp<"online_attention",
     AffineMap getValueMap() {
       return getIndexingMapsArray()[2];
     }
+    std::optional<AffineMap> getMaskMap() {
+      if (getMask()) {
+        return cast<AffineMap>(getIndexingMapsArray()[4]);
+      }
+      return std::nullopt;
+    }
     AffineMap getOutputMap() {
-      return getIndexingMapsArray()[3];
+      if (getMask()) {
+        return cast<AffineMap>(getIndexingMapsArray()[5]);
```

Comment:
Can we make the maps getter also do `getMask() ? 5 : 4` for outputMap and similarly for maxMap and sumMap?